docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: explain
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
```

Comment:
We could certainly use plain `EXPLAIN` in the short term, but long term I'd like `EXPLAIN` to show the query plan. The functionality described here seems primarily for internal use, hence the name `DEBUG`. Other options would be `EXPLAIN SCAN`, `EXPLAIN KEYS` or `EXPLAIN INTERNAL`.